hbase_sql/src/test/java/com/splicemachine/derby/utils/VacuumIT.java

```diff
@@ -85,7 +86,7 @@ public void testVacuumDoesNotBreakStuff() throws Exception {
             Set<String> deletedTables=getDeletedTables(beforeTables,afterTables);
             for(String t : deletedTables){
                 long conglom=new Long(t);
-                Assert.assertTrue(conglom>=DataDictionary.FIRST_USER_TABLE_NUMBER);
+                Assert.assertTrue(BasicDatabase.isSystemConglomerate(conglom));
```

Comment:
we need to change to negation here: !BasicDatabase.isSystemConglomerate(conglom)